app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.CAMERA" />
     <uses-permission android:name="android.permission.RECORD_AUDIO" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

Comment:
You can avoid this permission if you save it to a directory private to our app: https://developer.android.com/training/camera/photobasics#TaskPath